lib/zewo.rb

```diff
@@ -252,7 +226,7 @@ def uncommited_changes?(repo_name)
       end
 
       def master_branch?(repo_name)
-        name = `cd #{repo_name}; git rev-parse --abbrev-ref HEAD`
+        `cd #{repo_name}; git rev-parse --abbrev-ref HEAD`
       end
```

Comment:
Indeed, and you never use the name variable in this method. Besides that, Ruby returns always the last evaluated expression so you don't have to explicitly use 'return'.